### ISSUE TITLE
panOnScroll breaks onMoveEnd

### ISSUE THREAD

**Issue by frankandrobot** [wbkd/react-flow#3248]
### Describe the Bug

* When `panOnScroll` is set to false, `onMoveEnd` correctly fires whenever the user pans or zooms the graph. 
* however, once `panOnScroll is set to true, `onMoveEnd` no longer fires when the user pans or zooms the graph. It only fires when the user explictly clicks on the graph. 
*

### Steps to Reproduce the Bug or Issue

1.  Given this simple usage, enable `panOnScroll`

```ts
   <TopologyZoomProvider value={zoom}>
        <ReactFlow
          nodes={nodes}
          edges={edges}
          nodeTypes={reactFlowNodeTypes}
          edgeTypes={reactFlowEdgeTypes}
          fitView
          attributionPosition="top-right"
          onlyRenderVisibleElements
          onMoveEnd={(_, viewport) => {
            console.log("hello")
            setZoom(viewport.zoom);
          }}
          /** panOnScroll + selectionOnDrag makes it work like figma */
          panOnScroll
          selectionOnDrag
          minZoom={0.1}
        >
          <Controls />
        </ReactFlow>
```

### Expected behavior

* onMoveEnd should fire for any panning or zooming

### Platform

- OS: [macos]
- Browser: [firefox]
- Version: latest reactflow (as of today)

**Comment by moklick** [comment]
thanks I will check it 🧐

**Comment by moklick** [comment]
fixed in v11.8.0
